### DISCUSSION TITLE
dnscrypt-proxy local DoH server vs System default dnscrypt-proxy for using in Firefox

### DISCUSSION THREAD

**Discussion by khius** [DNSCrypt/dnscrypt-proxy#1925] [Q&A]
What are the differences? Which one is better?
<img width="675" alt="Screen Shot 2021-10-26 at 11 25 05 PM" src="https://user-images.githubusercontent.com/6244107/138994851-86412948-cf46-4bf6-b577-b42c4168c725.png">

**Comment by phantomcraft** [comment]
You are doing wrong.

You should enable the option "Enable DNS over HTTPS" just bellow the proxy configuration.

![Screenshot from 2021-10-27 20-41-16](https://user-images.githubusercontent.com/39815480/139167027-878e33ca-afd6-4dc8-834e-99ef21619d36.png)

**Reply by khius** [reply]
Ok, you mean that setting dnscrypt-proxy local DoH server in Firefox is better than using my system default dnscrypt-proxy DNS?

**Reply by phantomcraft** [reply]
It's good only if you want to use dnscrypt-proxy only in Firefox, it's your choice.

Here I use in entire system.

**Comment by jedisct1** [comment]
The local DoH server is just required if you want to beta test a Mozilla/Cloudflare experiment. 

It is far better to directly configure your system resolvers, as this will protect all your applications.
